pkg/virt-controller/watch/application.go

```diff
@@ -9,16 +9,31 @@ import (
 	"github.com/emicklei/go-restful"
 	clientrest "k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
 
 	"k8s.io/client-go/util/workqueue"
 
+	"k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/record"
+
+	"time"
+
 	kubeinformers "kubevirt.io/kubevirt/pkg/informers"
 	"kubevirt.io/kubevirt/pkg/kubecli"
 	"kubevirt.io/kubevirt/pkg/logging"
 	"kubevirt.io/kubevirt/pkg/virt-controller/rest"
 	"kubevirt.io/kubevirt/pkg/virt-controller/services"
 )
 
+const (
+	DefaultLeaseDuration = 15 * time.Second
```

Comment:
These look like sane defaults. I see k8s uses the same values as defaults.

We'll want this to be configurable at some point

Reply:
I thought about it, do we want to use k8s approach and to have additional parameters under `virt-controller`?
For example `virt-controller --leader-elect ...`